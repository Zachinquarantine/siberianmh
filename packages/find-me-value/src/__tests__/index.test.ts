import { findMeValue } from '../index'

describe('findMeValue()', () => {
  it('works', () => {
    const result = findMeValue('// Module Version: v1.0.0', 'Module Version')
    expect(result).toContain('v1.0.0')
    expect(result).toMatchSnapshot()
  })

  it('return null, if nothing finded', () => {
    const result = findMeValue('Anything', 'Module Naming')
    expect(result).toBe(null)
    expect(result).toMatchSnapshot()
  })

  it('throw error', () => {
    // @ts-ignore
    expect(() => findMeValue()).toThrowErrorMatchingSnapshot()
  })
})
