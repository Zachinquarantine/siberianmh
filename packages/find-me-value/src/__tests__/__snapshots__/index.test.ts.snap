// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`findMeValue() return null, if nothing finded 1`] = `null`;

exports[`findMeValue() throw error 1`] = `"Expected a string"`;

exports[`findMeValue() works 1`] = `" v1.0.0"`;
